README.adoc

```diff
@@ -141,53 +141,53 @@ include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.1.
-Import [hotspot=importMPApp file=1]`MicroProfileApplication` class and [hotspot=importContainer file=1]`Container` annotation, create the [hotspot=mpApp file=1]`MicroProfileApplication` application, and annotate with [hotspot=container file=1]`@Container`.
+Import the [hotspot=importMPApp file=1]`MicroProfileApplication` class and the [hotspot=importContainer file=1]`Container` annotation, create the [hotspot=mpApp file=1]`MicroProfileApplication` application, and annotate the application with [hotspot=container file=1]`@Container`.
 
 PersonServiceIT.java
 [source, java, linenums, role='code_column hide_tags=copyright,importInject,importAssertNotNull,importSharedContainerConfig,sharedContainerConfig,inject,personSvc,testCreatePerson']
 ----
 include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.2.java[]
 ----
 
-The [hotspot=withAppContextRoot file=1]`withAppContextRoot(String)` method indicates what the base path of the application is. In this case, our application is deployed at http://localhost:9080/guide-microshed-testing[http://localhost:9080/guide-microshed-testing^], so the app context root is [hotspot=withAppContextRoot file=1]`/guide-microshed-testing`, which is the portion of the URL after the hostname and port.
+The [hotspot=withAppContextRoot file=1]`withAppContextRoot(String)` method indicates the base path of the application. An app context root is the portion of the URL after the hostname and port. In this case, the application is deployed at the `http://localhost:9080/guide-microshed-testing` URL, so the app context root is [hotspot=withAppContextRoot file=1]`/guide-microshed-testing`.
 
-The [hotspot=withReadinessPath file=1]`withReadinessPath(String)` method indicates what path should be polled via HTTP to determine application readiness. MicroShed Testing will automatically start or discover the MicroProfileApplication and wait for it to be ready before the tests start running. In this case, we are utilizing the default application readiness check at http://localhost:9080/health/ready[http://localhost:9080/health/ready^], which is provided by the [hotspot=mpHealth file=2]`mpHealth-2.0` feature in our server.xml configuration. When the readiness URL returns HTTP 200, the application is considered ready and the tests will begin running.
+The [hotspot=withReadinessPath file=1]`withReadinessPath(String)` method indicates what path is polled by HTTP to determine application readiness. MicroShed Testing automatically starts the MicroProfileApplication application and waits for it to be ready before the tests start running. In this case, we are using the default application readiness check at the http://localhost:9080/health/ready[http://localhost:9080/health/ready^] URL, which is enabled by the [hotspot=mpHealth file=2]`mpHealth-2.0` feature in our server.xml configuration file. When the readiness URL returns `HTTP 200`, the application is considered ready and the tests begin running.
```

Comment:
Change
"In this case, we are using the default application readiness check at the http://localhost:9080/health/ready URL, which is enabled by the mpHealth-2.0 feature in our server.xml configuration file."
To
"In this case, you are using the default application readiness check at the http://localhost:9080/health/ready URL, which is enabled by the mpHealth-2.0 feature in the server.xml configuration file."

Reply:
addressed